lte/gateway/deploy/roles/agw_docker/vars/all.yaml

```diff
@@ -23,7 +23,7 @@ magma_use_pkgrepo: yes
 
 docker_compose_version: 1.29.2
 
-docker_registry: ""
+docker_registry: "public.ecr.aws/z2g3r6f7/"
```

Comment:
Authentication is required here. How do we ensure it?
Should we maybe consider to push it to GHCR and make it public without the need of credentials?

Reply:
this a public repo and does not authentication require any authentication.
you need to pull the images and not just open the url.

Reply:
Aah, ok, my bet.
Which ones would be the images to be pulled here?
Is there any UI to browse this Docker-Repo?
Or any other method to verify this link?

> Also this not an official build but rather a configuration item to be change.

I do not quite understand this sentence. What do you mean by that?

Reply:
Yes, you can check https://gallery.ecr.aws/z2g3r6f7/
Or pull the images directly here,
docker pull public.ecr.aws/z2g3r6f7/agw_gateway_python:latest
docker pull public.ecr.aws/z2g3r6f7/agw_gateway_c:latest

> I do not quite understand this sentence. What do you mean by that?
This is a build I generated myself and that people can use for testing purpose. We don't have yet an official release process for containerized agw since It is still in alpha phase.

Reply:
> This is a build I generated myself and that people can use for testing purpose.

Persisting personal testing images in the main Magma code might not be a good idea. We might consider to create a official imagestream on GHCR for those images instead.